oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxHWDiskStore.java

```diff
@@ -66,6 +67,15 @@
     private static final String ID_MODEL = "ID_MODEL";
     private static final String ID_SERIAL_SHORT = "ID_SERIAL_SHORT";
 
+    private static final String DM_UUID = "DM_UUID";
+    private static final String DM_VG_NAME = "DM_VG_NAME";
+    private static final String DM_LV_NAME = "DM_LV_NAME";
+    private static final String LOGICAL_VOLUME_GROUP = "Logical Volume Group";
+    private static final String DEV_NAME = "DEV_NAME";
+    private static final String DEV_PATH = "DEV_PATH";
+    private static final String DEV_LOCATION = "/dev";
+    private static final String DEV_MAPPER = DEV_LOCATION + "/mapper";
```

Comment:
Similarly, add slash after mapper and remove before if you've added it to DEV_LOCATION....

Reply:
ok